sagemaker-neo-notebooks/edge/windows/README.md

```diff
@@ -0,0 +1,216 @@
+
+# C++ Tutorial
+
+## Prerequisite
+
+Please review the following prerequisites tasks before proceed with tutorial
+
+### Environment Setup
+
+#### Environment Setup for Windows
+
+In order to compile DLR and tutorial dependencies, please install [Visual Studio 2019](https://visualstudio.microsoft.com/vs/) and [WinML](https://docs.microsoft.com/en-us/windows/ai/windows-ml/get-started-desktop) components. You also need to install [CppWinRT package](https://marketplace.visualstudio.com/items?itemName=CppWinRTTeam.cppwinrt101804264). 
```

Comment:
I think WinML and CppWinRT are not needed for this sample and can be removed.

Reply:
note taken.